_articles/participate.md

```diff
@@ -2,24 +2,52 @@
 layout: participate
 title: Attend an Event
 ---
-
 <div class="content-box content-box--full" markdown="1">
+  <h3 class="content-title">
+    <span class="title-frame"></span>
+    Steps on how to sign up for a local event
+    <span class="title-frame title-frame--rotate-180"></span>
+  </h3> 
+    
+  <p>1. Look for an event in your area</p>
+
+  {% include map.html %}
+  
+  <p>2. Book your seat on the signup page.</p>
+  <p>3. If there is no event in your area, consider <a href="{{ '/host/' | prepend: site.baseurl }}">hosting</a> one yourself :)</p>
+
+</div>
+
+<h3 class="content-title">
+  <span class="title-frame-grey"></span>
+  Do the action solo
+  <span class="title-frame-grey title-frame--rotate-180"></span>
+</h3> 
+
+<p class="participate-theaser">You prefer to participate at the campaign from the comfort of your couch? That’s totally fine! Here is how:</p>
+
+<section class="info-buttons">
+  <div class="nightly">
+    <a href="{{ '/reporting/' | prepend: site.baseurl }}"><img src="/img/beta-logo.png" alt="" /></a>
+    <p>
+      Install <a href="https://www.mozilla.org/en-US/firefox/channel/desktop/#beta">Firefox Beta</a> for Desktop
+    </p>
+  </div>
+  <div class="alexa">
+    <a href="{{ '/leaderboard/' | prepend: site.baseurl }}"><img src="/img/icon-color-browser.svg" alt="" /></a>
+    <p>
+      Check the <a href="https://www.alexa.com/topsites/countries">Alexa list</a> of the most famous websites in your local area and choose the website you want to test. Please don't start at the top to make sure we do not only test the top 10.
+    </p>
+  </div>
+  <div class="report-attend">
+    <a href="https://github.com/{{ site.github.repo }}"><img src="/img/icon-color-report.svg" alt="" /></a>
+    <p>
+      Once you have finished testing a website go and <a href="{{ '/reporting/' | prepend: site.baseurl }}">report it on the report form</a>, you will find all the info you need on how to report your work
+    </p>
+  </div>
+</section>  
+
+Keep doing the awesome work and make Firefox better!
+
+<p class="help">Help us improve content and <a herf="https://github.com/mozilla/firefox57-sprint/">suggest changes</a> on this page.</p>
```

Comment:
We have removed this from the other pages (still in the mockups though, thanks for that attention to detail!). Would you mind removing this?